### PR TITLE
Initial config for `help-desk-service`.

### PR DESCRIPTION
**Pull request by NickFitz**

### CODE REVIEW

File: help-desk-service.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    type: gds
+    region: eu-west-2
+    app: dit-staging/sre-ticketing-dev/help-desk-service-staging
+    vars: []
+    secrets: true
+    run: []
+  - environment: prod
+    type: gds
+    region: eu-west-2
+    app: dit-services/sre-ticketing/help-desk-service-prod
```
Comment (ejayesh) [inline help-desk-service.yaml:16]: This should be:
app: dit-services/sre-ticketing/help-desk-service

File: help-desk-service.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+name: help-desk-service
+namespace: help-desk-service
+scm: git@github.com:uktrade/help-desk-service.git
+environments:
+  - environment: staging
+    type: gds
+    region: eu-west-2
+    app: dit-staging/sre-ticketing-dev/help-desk-service-staging
```
Comment (ejayesh) [inline help-desk-service.yaml:9]: this should be:
    app: dit-staging/sre-ticketing-staging/help-desk-service-staging

**Review by ejayesh — CHANGES_REQUESTED** [review]
A couple of suggestions, see comments

File: help-desk-service.yaml (hunk 2 above)
Comment (NickFitz) [reply help-desk-service.yaml]: Just FYI, the existing space is called `sre-ticketing-dev` but has a staging app deployed to it.

**Review by ejayesh — APPROVED** [review]
lgtm

**Pull request merged**